src/themes/clean/templates/journal/full-text-search.html

```diff
@@ -38,17 +38,23 @@ <h1>{% trans 'Article Search Results' %}</h1>
                                 {% include 'elements/forms/errors.html'  with form=form%}
                                 {% bootstrap_form form %}
                             {% else %}
-                                {% trans "You are currently browsing by keyword" %}. <a href="{% url 'search' %}">{% trans 'Search for an article' %}.</a>
+                                {% trans "You are currently browsing by keyword." %} 
```

Comment:
If changing the key to this translation target we need to:
 - Update the key in the translation files
 - Update the translations so they match the new default punctuation